bidwire/bidwire_settings.py

```diff
@@ -19,7 +21,7 @@
 ADMIN_EMAIL = "bidwire-admin@googlegroups.com"
 
 
-def get_recipients_list(env_var_name, default_recipients='bidwire-logs@googlegroups.com'):
+def get_recipients_list(env_var_name, default_recipients='zacharyspector@gmail.com'):
```

Comment:
The default should not be your personal email.

You can do your local testing without having to modify any of the defaults or settings, by using the `manage.py` script. For example, to run the scraper and the notifier for your site, you can do:
```
python bidwire/manage.py dryrun --site <your site enum> --recipients <your email>
```

There are other examples in the `README`.